WPMedia/fusion-news-theme-blocks
# fix: exception in links-bar-block

## Description

The links-bar-block is throwing a pagebuilder console error.

## Jira Ticket

- [THEMES-1259](https://arcpublishing.atlassian.net/browse/THEMES-1259)

## Acceptance Criteria

This should not throw an error and should be updated to one of the acceptable values.

## Test Steps

1. Checkout this branch `git checkout THEMES-1259`
2. Run storybook and see the console.

## Effect Of Changes

### Before

This throws an error in the pagebuilder console.

### After

No error should be thrown

## Dependencies or Side Effects

- Feature pack pr for local development:

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1259]: https://arcpublishing.atlassian.net/browse/THEMES-1259?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[THEMES-1259]: https://arcpublishing.atlassian.net/browse/THEMES-1259?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
